public/app/features/explore/LogsContainer.tsx

```diff
@@ -195,16 +167,16 @@ function mapStateToProps(state: StoreState, { exploreId }: { exploreId: string }
     absoluteRange,
     dedupStrategy,
   } = item;
-  const dedupedRows = deduplicatedRowsSelector(item);
+  const dedupedRows = deduplicatedRowsSelector(item) || undefined;
   const timeZone = getTimeZone(state.user);
 
   return {
     loading,
     logsHighlighterExpressions,
-    logRows: logsResult && logsResult.rows,
-    logsMeta: logsResult && logsResult.meta,
-    logsSeries: logsResult && logsResult.series,
-    visibleRange: logsResult && logsResult.visibleRange,
+    logRows: logsResult?.rows || undefined,
```

Comment:
You don't need to add || undefined here. [See](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)
> The ?. operator functions similarly to the . chaining operator, except that instead of causing an error if a reference is nullish (null or undefined), the expression short-circuits with a return value of undefined.